docs/preview.md

```diff
@@ -43,11 +51,17 @@ To enable preview style formatting without enabling any preview lint rules:
     preview = true
     ```
 
+=== "CLI"
+
+    ```console
+    ruff format . --preview
+    ```
+
 ## Using rules that are in preview
 
 If a rule is marked as preview, it can only be selected if preview mode is enabled. For example, consider a
 hypothetical rule, `HYP001`. If `HYP001` were in preview, it would _not_ be enabled by adding following to your
-config file:
+config file or only using the `--select` flag on the CLI:
```

Comment:
If you want to add more here, I'd rephrase entirely

> If `HYP001` were in preview, it would _not_ be enabled by adding it to the selected rules.

Reply:
Done in 9b6b7bd19.